recipes/user.rb

```diff
@@ -13,6 +13,7 @@
 
 user conf['user'] do
   home conf.home
+  group conf.group
```

Comment:
Those don't apply since we do `conf = node['stackstorm']['user']`. See:

```
       Deprecated features used!
         method access to node attributes (node.foo.bar) is deprecated and will be removed in Chef 13, please use bracket syntax (node["foo"]["bar"]) at 3 locations:
           - /tmp/kitchen/cache/cookbooks/yum-erlang_solutions/attributes/erlang_solutions.rb:1:in `from_file'
           - /tmp/kitchen/cache/cookbooks/yum-erlang_solutions/attributes/erlang_solutions.rb:2:in `from_file'
           - /tmp/kitchen/cache/cookbooks/mongodb3/recipes/package_repo.rb:26:in `from_file'
         node.set is deprecated and will be removed in Chef 14, please use node.default/node.override (or node.normal only if you really need persistence) at 2 locations:
```

which applies to the `yum-erlang_solutions` and `mongodb3` cookbooks.